format/Message.fbs

```diff
@@ -159,26 +173,30 @@ struct FieldNode {
   /// to write their physical validity bitmap out as a materialized buffer,
   /// instead setting the length of the bitmap buffer to 0.
   null_count: int;
+
+  /// children according to the schema
+  children: [FieldNode];
+
+  /// Buffers correspond to the pre-ordered flattened buffer tree
+  ///
+  /// The number of buffers appended to this list depends on the field type and length.
+  /// For example, most primitive arrays will have 2 buffers, 1 for the validity
+  /// bitmap and 1 for the values. For struct arrays, there will only be a
+  /// single buffer for the validity (nulls) bitmap
+  vectors: [Buffer];
```

Comment:
Ah right. I keep calling them vectors as in "Vectorized Execution". the content of the buffer is in a Buffer.
I think Vector is more specific but I'm open to other views :)